open-sphere-plugins/my-places/src/main/java/io/opensphere/myplaces/controllers/MyPlacesDataGroupController.java

```diff
@@ -121,7 +121,7 @@ public void loadDataGroups()
                 folder.setVisibility(Boolean.TRUE);
 
                 dataGroup = new MyPlacesDataGroupInfo(true, myToolbox, folder);
-                dataGroup.setIsFlattenable(true);
+                dataGroup.setIsFlattenable(false);

```

Comment:
Is this necessary if the `MyPlacesDataGroupInfo` constructor does the same thing?